src/libs/fsdrv/lv_fs_littlefs.c

```diff
@@ -0,0 +1,165 @@
+#include "../../../lvgl.h"
+#if LV_USE_FS_LITTLEFS != '\0'
+
+#include <LittleFS.h>
+#include "../../core/lv_global.h"
```

Comment:
where is LittleFS.h ?
need 3rd library support?

Reply:
Yes it's based on: https://github.com/esp8266/Arduino/blob/master/libraries/LittleFS/src/LittleFS.h
I don't know if there is another available implementation

Reply:
we should not use and refer Arduino related code in LVGL, because LVGL can be used with other OS / framework, too. For example with FreeRTOS, unix, MicroPython, etc…

If we want to use a 3rd party library, then refer to (or copy) it’s original (and 
possibly platform-independent) code. 
In case of LittleFS: https://github.com/littlefs-project/littlefs

For example in MicroPython framework they use the original code: https://github.com/micropython/micropython/tree/master/lib/littlefs

Reply:
I'm gonna use the littlefs project you linked! You're totally right about the Arduino, didn't know what's the process for this kind of dependency

Reply:
Woot, implementing the based lfs.h library is harder than expected :sweat_smile: 
I'm struggling with platformio:
```
.pio/libdeps/m5stack-core2/lfs/runners/test_runner.c:23:10: fatal error: execinfo.h: No such file or directory

******************************************************************
* Looking for execinfo.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:execinfo.h"
* Web  > https://registry.platformio.org/search?q=header:execinfo.h
*
******************************************************************

 #include <execinfo.h>
          ^~~~~~~~~~~~
compilation terminated.
```

Also, is there a way to declare custom lfs configuration? I want to make lfs_config more generic and don't know if it should be configured in lv_conf, or elsewhere :thinking: 
```c
static void fs_init(void)
{
  const struct lfs_config cfg = {};

  lfs_mount(&lfs, &cfg);
}
```

Because of my M5Stack Core2 configuration, I think I have to backport this configuration to be sure everything is working well with the new implementation, I would like to reuse the same configuration as it's defined in this file
https://github.com/esp8266/Arduino/blob/master/libraries/LittleFS/src/LittleFS.h#L62


At least, I update the code until I find a way to make it work on my project ^^

Reply:
I can't find an easy way to use what's defined in Arduino's configuration, everything is private/protected :(
If we assume that LVGL should not be aware of the configuration, maybe in a first step we can use the "LittleFS.h" dependency ?
Regarding this [comment](https://github.com/lvgl/lvgl/pull/5562#discussion_r1477779716)  what do you mean by `lv_littlefs_set_handler`? Is there an example somewhere?